lib/PuppeteerSharp.Tests/PuppeteerBrowserBaseTest.cs

```diff
@@ -18,16 +18,15 @@ public PuppeteerBrowserBaseTest(ITestOutputHelper output) : base(output)
             {
                 dirInfo.Create();
             }
-
-            InitializeAsync().GetAwaiter().GetResult();
         }
 
-        protected virtual async Task InitializeAsync()
-        {
+        Task IAsyncLifetime.InitializeAsync() => InitializeAsync();
```

Comment:
So, we don't need this because `protected virtual async Task InitializeAsync() =>` is implicitly implementing the interface (the same for DisposeAsync)

Reply:
I didn't want to change visibility of InitializeAsync from protected to public just yet. Happy to do so now.

Reply:
Don't forget XML comments when you change the scope :)